stores/governance/src/main/java/com/bloxbean/cardano/yaci/store/governance/domain/CommitteeRegistration.java

```diff
@@ -0,0 +1,28 @@
+package com.bloxbean.cardano.yaci.store.governance.domain;
+
+import com.bloxbean.cardano.yaci.store.common.domain.BlockAwareDomain;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class CommitteeRegistration extends BlockAwareDomain {
+    private String txHash;
+
+    private long certIndex;
+
+    private Long slot;
```

Comment:
Please also add epoch to every domain model/entity, as the epoch is a key attribute for the ratification and enactment of governance proposals.